### PR TITLE
Userservices

### PR DESCRIPTION
**Pull request by PratibhaSaini2601**

### CODE REVIEW

File: src/main/java/com/wcc/cricket/controller/UserController.java [hunk 1]
```diff
@@ -25,8 +30,18 @@ public ResponseEntity<UserResponse> getUserByID(@PathVariable int userID) {
         return ResponseEntity.ok(userService.getUserByID(userID));
     }
```
Comment (PratibhaSaini2601) [inline src/main/java/com/wcc/cricket/controller/UserController.java:32]: should the users controller allow you add multiple user together i.e. should the argument be List<RegisterRequest>? In that case what would we return in the response body List of JWTs?

Comment (abhijeetsingh26) [reply src/main/java/com/wcc/cricket/controller/UserController.java:32]: Does not make sense for a single user to be able to add multiple users. IMO this should be more of an admin task. A new Endpoint which can take a list of users to be created and returns the what users were successfully created. In this flow there is no need to returning multuiple JWTs for each user.

**Comment by abhijeetsingh26** [comment]
- [ ] Please delete the branch after successfull merge.

**Comment by PratibhaSaini2601** [comment]
I have delete userService. projectRename might get some more changes pushed into it.

**Pull request merged**